bot.rb

```diff
@@ -31,8 +31,8 @@
         when "/help"
           send.(HELLO)
         when "/story"
-          current_step = 0
-          check = true
+          CURENT_STEP = 0
+          CHECK = true
```

Comment:
Sorry for long time response. I try to make my bot working with inline buttons (m.b. you see prototype in bot_2.rb) Hope to finish it tomorrow